lib/ontologies_linked_data/serializers/json.rb

```diff
@@ -6,49 +6,96 @@ class JSON
       CONTEXTS = {}
 
       def self.serialize(obj, options = {})
+        # Handle mod object serialization apart to not break everything
+        return serialize_mod_objects(obj, options) if mod_object?(obj)
 
+        # Handle the serialization for all other objects in the old way
```

Comment:
replace the old way by standard way,
and a question why MOD object serialization is different of the serialization of current objects ?

Reply:
the serialization of MOD objects is different to not break other routes and models, like ontologies. when we want to migrate ontologies for example to hydra pagination we have i think to update other things in API and in the UI